src/main/kotlin/racingcar/domain/record/CarRecord.kt

```diff
@@ -1,8 +1,5 @@
 package racingcar.domain.record
 
-import racingcar.domain.car.Car
-
 data class CarRecord(
-    val car: Car,
-    val records: List<Int> // Car의 바퀴마다 position을 저장
+    var record: List<Int>
```

Comment:
```suggestion
    val record: List<Int>
```
지금과 같이 외부에서 접근할 수 있는 var을 사용하게 되면, 외부 클래스에서 데이터 조작이 가능합니다.
그렇게 되면, record를 참조하고 있는 곳에서는 예상치 못한 변경으로 인하여 오류가 발생할 가능성이 높습니다.
var 대신 val을 활용해보면 어떨까요?

val로 변경하게 되면, 외부에서 값을 변경하지 못하게 될텐데요.
이런 경우에는 값이 변경될 때마다 객체를 매번 새로 만들어주는 방법이 있습니다.
data class는 copy 함수를 제공하고 있습니다.
이참에 copy 함수는 어떻게 복사를 하는지 학습해보시면 좋을 것 같습니다.
추가로 List에 새 값을 추가할 때에는 plus라는 함수를 활용하시면 됩니다. 😄

Reply:
val로 변경하고
`carRecord.copy(record = carRecord.record.plus(position))`와 같이 처리했습니다!